regulations/templates/regulations/base.html

```diff
@@ -44,8 +46,6 @@
         <!--[if lt IE 9]>
             <script src="{%static "regulations/js/built/lib/respond/respond.min.js" %}"></script>
         <![endif]-->
-        {% block preload %}
-        {% endblock %}
```

Comment:
This change is no longer needed, hey? I don't know if anything depends on this order.